webui/react/src/contexts/AppContexts.tsx

```diff
@@ -62,7 +61,14 @@ const AppContexts: React.FC = () => {
     triggerNotebooksRequest({});
     triggerShellsRequest({});
     triggerTensorboardsRequest({});
-    triggerActiveExperimentsRequest({ states: activeRunStates });
+    triggerActiveExperimentsRequest({
+      states: [
+        'STATE_ACTIVE',
+        'STATE_STOPPING_CANCELED',
+        'STATE_STOPPING_COMPLETED',
+        'STATE_STOPPING_ERROR',
+      ],
```

Comment:
**question:** did `activeRunStates` not work for you?

Reply:
Those are the TS doubts I have, the 2 values are different:
- Api is expecting a `'STATE_ACTIVE'`
- activeRunStates.Active = `'ACTIVE'`

How do you think is best to handle that `STATE_` difference?

Reply:
Ahhh I see... this is because of the definition differences between the output and the input states...
I think the best thing to do is just to replace the definition of `activeRunStates` to be (in case we want to reuse it elsewhere):
```js
export const activeRunStates = [
  'STATE_ACTIVE',
  'STATE_STOPPING_CANCELED',
  'STATE_STOPPING_COMPLETED',
  'STATE_STOPPING_ERROR',
];
```